sphinx_docs/conf.py

```diff
@@ -1,14 +1,12 @@
 """Configure sphinx for package doc publication."""
 import os
+from pathlib import Path
 
 import tomlkit
 
 PYPROJECT_FILE = "pyproject.toml"
 EXPLANATION = "- needed to extract version information."
-file_path = os.path.join(
-    os.path.abspath(os.path.join(__file__, os.path.pardir, os.path.pardir)),
-    PYPROJECT_FILE,
-)
+file_path = Path(__file__).parents[1] / PYPROJECT_FILE
```

Comment:
So maybe this is fine, but it looks like we are dividing a string by another string?  Could just be the diff or me being bad at life.

Reply:
Yeah so I tried it, and it works.  I guess there is some black magic with the PosixPath object that allows a division operator to basically act as a join.